src/System.Data.SqlClient/src/System/Data/SqlClient/SqlParameter.cs

```diff
@@ -1640,6 +1676,11 @@ internal void SetUdtLoadError(Exception e)
             _udtLoadError = e;
         }
 
+        private void Set(SqlParameterFlags flag, bool value)
+        {
+            _flags = value ? _flags | flag : _flags & ~flag;
```

Comment:
I wonder whether this could be easier to read broken out a bit.

Reply:
It could be, i've used the pattern elsewhere to combine flags but have no objection to changing it. It was promoted to a function from being inline in multiple locations which is why it's iif-style.

Reply:
Done